examples/teleop_ros2/python/teleop_ros2_publisher.py

```
@@ -182,184 +181,226 @@ def _append_hand_poses(
-                search_paths=plugin_paths,
+    def __init__(self) -> None:
+        super().__init__("teleop_ros2_publisher")
+
+        self.declare_parameter("hand_topic", "xr_teleop/hand")
+        self.declare_parameter("twist_topic", "xr_teleop/root_twist")
+        self.declare_parameter("pose_topic", "xr_teleop/root_pose")
+        self.declare_parameter("controller_topic", "xr_teleop/controller_data")
+        self.declare_parameter("full_body_topic", "xr_teleop/full_body")
+        self.declare_parameter("frame_id", "world")
+        self.declare_parameter("rate_hz", 60.0)
+        self.declare_parameter("use_mock_operators", False)
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)